src/redux/categories/categories.js

```diff
@@ -0,0 +1,11 @@
+const Status = 'bookstore/categories/Status';
+const initialState = { categories: [] };
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case Status:
+      return { ...state, categories: action.payload };
+    default:
+      return state;
```

Comment:
- [ ] as part of the requirements, in case action.type is CHECK_STATUS the function returns a string "Under construction". kindly fix that 🙏